time stamp
int32 dataPub1
int32 dataPub2